test/extension/load_extension.test

```diff
@@ -12,14 +12,17 @@ PRAGMA enable_verification
 statement error
 LOAD 'asdf';
 ----
+<REGEX>:.*IO Error: Extension.*not found.*
```

Comment:
Just curious more than anything, why do we need `.*` at the start, should it not start with `IO Error:` ?

Reply:
Good question, I thought about the case, when the string describing the Error type changes, but still will contain "IO Error" in it, so we don't have to fix all the tests testing this error type showing up. 
But I'm not sure that if it will be the case. I'll remove `.*` from the beginning of the expected error messages in the test in further commit.